src/components/Menu/Menu/hooks/useLastNavigationDirection.jsx

```diff
@@ -0,0 +1,33 @@
+import { useRef, useState, useCallback } from "react";
+import {
+  ARROW_DOWN_KEYS,
+  ARROW_LEFT_KEYS,
+  ARROW_RIGHT_KEYS,
+  ARROW_UP_KEYS,
+  NAV_DIRECTIONS
+} from "../../../../hooks/useFullKeyboardListeners";
+import useKeyEvent from "../../../../hooks/useKeyEvent";
+
+const NAVIGATION_KEYS = [...ARROW_UP_KEYS, ...ARROW_RIGHT_KEYS, ...ARROW_DOWN_KEYS, ...ARROW_LEFT_KEYS];
+
+export const useLastNavigationDirection = () => {
```

Comment:
This will be used in the new `MenuGridItem`, to detect the direction of the navigation.
Basically I can use it in more existing places and save some code and complexity - maybe I'll do it when I have some time